Gemfile

```diff
@@ -82,7 +82,7 @@ group :development, :test do
   gem 'rspec-rails', '~> 3.2'
   gem 'capybara', '~> 2.5'
   gem 'database_cleaner'
-  gem 'factory_girl_rails'
+  gem 'factory_girl_rails', require: false
```

Comment:
Es gab einen Fehler bei den Migrations in der wimi-Factory, weil er dort versucht hat, auf die chair-Factory zuzugreifen, obwohl dieser ja erst viel später kam. Das war ein kurzzeitiger Fix dafür (Es wurde ja trotzdem weiterhin verwendet, deshalb die Zusatzzeile im spec_helper).
Wir haben das jetzt aber auf andere Weise behoben (bei der Zuweisung des chairs in der Factory '{}' um den value).